cms/djangoapps/import_from_modulestore/models.py

```diff
@@ -14,25 +13,20 @@
     UsageKeyField,
 )
 from openedx_learning.api.authoring_models import LearningPackage, PublishableEntity
+from user_tasks.models import UserTaskStatus
 
 from .data import ImportStatus
 
 User = get_user_model()
 
 
-class Import(TimeStampedModel):
+class Import(models.Model):
```

Comment:
Shouldn't we add a `composition_level` and `override` fields to this model? (Based on this [PR](https://github.com/openedx/edx-platform/pull/36582/files#diff-660aa8c549956602b8f81c77c2f7c0914a8ec1b6baf646de66d72981be35aeceR45) from Kyle's feedback?)